hugegraph-core/src/main/java/com/baidu/hugegraph/util/ConfigUtil.java

```diff
@@ -96,18 +121,17 @@ public static void writeToFile(String dir, String graphName,
         E.checkArgument(FileUtils.getFile(dir).exists(),
                         "The directory '%s' must exist", dir);
         String fileName = Paths.get(dir, graphName + CONF_SUFFIX).toString();
-        try (OutputStream os = new FileOutputStream(fileName)) {
-            config.save(os, CHARSET);
-            config.setFileName(fileName);
+        try {
+            config.save(new File(fileName));
```

Comment:
try to run client api test to check it work?

Reply:
are you mean "hugegraph-client" unit test?
if not, I have run hugegraph-test/api test in local and it is run ok

Reply:
yes, hugegraph-client api test may go here when dynamic creating graph

Reply:
ok

Reply:
I have run hugegraph-test successfull on local